presidio-analyzer/conf/transformers.yaml

```diff
@@ -4,4 +4,41 @@ models:
     lang_code: en
     model_name:
       spacy: en_core_web_sm
-      transformers: elastic/distilbert-base-uncased-finetuned-conll03-english
+      transformers: StanfordAIMI/stanford-deidentifier-base
+
+ner_model_configuration:
+  labels_to_ignore:
+  - O
+  aggregation_strategy: simple # "simple", "first", "average", "max"
+  stride: 16  # If stride >= 0, process long texts in
+              # overlapping windows of the model max
+             # length. The value is the length of the

```

Comment:
**nit**: Spacing for start of comment is different for above lines and below lines.

Reply:
Done